src/lib/fascist.c

```diff
@@ -5,6 +5,10 @@
  */
 
 #include "config.h"
+#ifdef HAVE_GETPWUID_R
+// Needed for LINE_MAX
+#define	_XOPEN_SOURCE 700
```

Comment:
we should set the standard min in configure.ac rather than sprinkle it throughout the tree

Reply:
Removed. This is actually not needed, was caused by the `-std=c99` compiler option used when testing.

Reply:
we should be selecting a language standard too at this point. I'd be fine with C17, but if someone comes out of the woodwork, I'd settle for C11. anything older is unreasonable.